main/editor.js

```diff
@@ -41,7 +66,9 @@ const openEditorWindow = async (filePath, recordFps) => {
 
   loadRoute(editorWindow, 'editor');
 
+  editorWindow.on('close', discardVideo);
   editorWindow.on('closed', () => {
+    editorWindow = null;
```

Comment:
Good catch, Fix is coming!

Reply:
Yeah, since it'll be different functions when adding/removing the event listener I guess it's ok. I don't know a way to remove the event listener inside a function that's called with arguments without doing some dirty hacks.